spec/lib/jobs/install-os-job-spec.js

```diff
@@ -278,4 +278,82 @@ describe('Install OS Job', function () {
             });
         });
     });
+
+    describe('test _validateOptions', function() {
```

Comment:
It's better to add following test case for better coverage:
- If uid is not given, the options validation will still pass.
- If sshKey is not given, the options validation will still pass.